### ISSUE TITLE
We should configure SQL dumps for all MySQL and PostgreSQL databases

### ISSUE THREAD

**Issue by jimmybergman** [atomia/puppet-atomia#24]

**Comment by nbeernink** [comment]
I think you can simply use https://github.com/puppetlabs/puppetlabs-mysql#mysqlserverbackup and https://github.com/puppetlabs/puppetlabs-postgresql#create-recovery-configuration for this. No need to re-invent the wheel. The classes are pretty complete and easy to use. Perhaps they weren't around yet in 2014? :)

**Comment by stefanmortensen** [comment]
Yes agreed, the current standpoint is to use already created modules if they exist :)
